.semaphore/semaphore.yml

```diff
@@ -0,0 +1,57 @@
+version: v1.0
+name: confluent-docker-utils
+agent:
+  machine:
+    type: s1-prod-ubuntu20-04-amd64-1
+execution_time_limit:
+  minutes: 30
+global_job_config:
+  prologue:
+    commands:
+      - checkout
+      - sem-version python 3.8
+      - pip install tox
+      - COMMIT_MESSAGE_PREFIX="Publish version"
+      - LAST_COMMIT_MESSAGE=$(git log -1 --oneline --format="%ae %s")
+      - >- 
+        if [[ "$LAST_COMMIT_MESSAGE" =~ "semaphore@confluent.io $COMMIT_MESSAGE_PREFIX" ]]; then
```

Comment:
If you include `[ci skip]` in the commit message, then Semaphore will ignore the commits